### ISSUE TITLE
Crashed in Demo mode

### ISSUE THREAD

**Issue by JPT77** [a2-4am/4cade#552]
Hi, 

when you don't touch anything it switches to demo mode, correct?

I had it run for a while and it crashed with this screenshot shown:
![IMG_20230603_194914](https://github.com/a2-4am/4cade/assets/4168125/70c087d9-73ce-4e6a-be10-1ffca0ee14d0)

so I assume there is a bug that makes the demo crash?
No idea which game this is. Google picture search did not find a similar screenshot.

It could be that it crashed because of hardware failure, but I am pretty sure my Apple is now rather stable. 
Hardware: Apple ][europlus with only Apple2IORpi for virtual harddrive and Language card.  (it is colored because I use the HDMItoRGB which is able to guess color even for PAL apple)
Total Replay version 5

**Comment by frankmilliron** [comment]
Hi. We suspect that there is an issue with the DRAM on your machine. Can you try running the Apple II+ Dealer Diagnostics "Motherboard RAM Test" and "Card Tests-->Language Card"? It can tell you exactly which chip has failed.
https://mirrors.apple2.org.za/ftp.apple.asimov.net/images/disk_utils/diagnostics/Apple%20II%2B%20Dealer%20Diagnostics.dsk

**Comment by JPT77** [comment]
Well, I checked the RAM a lot of times already, seems fine. 
but yes, something is instable. I had the demo running for about two hours and every 20 minutes it crashed into monitor.
so in this case let's close this issue. 

ps. My plan is to buy the RetroChipTester (RCT) and check all chips. I hope the RCT will identify chips with weak signals as well. And I will try replacing easy available chips with new ones and solder them to the board.

We already fixed so much issues with this board and still it's not stable.
